FastSimulation/Tracking/python/PixelLessTracking_cff.py

```diff
@@ -10,7 +10,7 @@
 pixelLessWithMaterialTracks = RecoTracker.TrackProducer.CTFFinalFitWithMaterial_cfi.ctfWithMaterialTracks.clone()
 # The sequence
 pixelLessTracking = cms.Sequence(pixelLessSeeds*pixelLessTrackCandidates*pixelLessWithMaterialTracks)
-pixelLessTrackCandidates.SeedProducer = cms.InputTag("pixelLessSeeds","PixelLess")
+pixelLessTrackCandidates.SeedProducer = cms.InputTag("pixelLessSeeds")
 pixelLessWithMaterialTracks.src = 'pixelLessTrackCandidates'
 pixelLessWithMaterialTracks.TTRHBuilder = 'WithoutRefit'
 pixelLessWithMaterialTracks.Fitter = 'KFFittingSmootherWithOutlierRejection'
```

Comment:
is there a better name for this file? It is used in HLT if I am not mistaken... The current name confuses me with PixelLessStep_cff.py